infra/testing/sw-env.js

```diff
@@ -24,6 +24,17 @@ const swEnv = makeServiceWorkerEnv();
 // This is needed to ensure new URL('/', location), works.
 swEnv.location = 'https://example.com';
 
+// See https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent/FetchEvent
+class FetchEvent {
```

Comment:
Nit: could you move this out into mock-fetch-event.js and then import the FetchEvent class and add it to global or swEnv. Just helps seperate the mock logic from what mocks are actually being used.

Reply:
I've switched to using https://github.com/pinterest/service-workers/pull/50